File: truyenvkl/src/page.lua
local url = ...
local doc = http:get(url):html()
local list = {}
local pages = doc:select(".pagination li")
local page = 1
local listChap = doc:select("#dsc")
if not listChap:isEmpty() then
    if pages:size() > 1 then
        page = num:to_int(regexp:find(pages:get(pages:size() - 2):select("a"):attr("href"), "/(\\d+)"), 1)
    end
    for i = 1, page do
        table.insert(list, url .. "/" .. i)
    end
else
    local bookId = doc:select("#views"):attr("data-id")
    table.insert(list, bookId)
end

return response:success(list)

File: truyenvkl/src/page.js
function execute(url) {
    var doc = Http.get(url).html();
    var list = [];
    var pages = doc.select(".pagination li");
    var page = 1
    var listChap = doc.select("#dsc");
    if (!listChap.isEmpty()) {
        if (pages.size() > 1) {
            page = pages.get(pages.size() - 2).select("a").attr("href").match(/\/(\d+)/);
            if (page) page = parseInt(page[1]); else page = 1;
        }
        for (var i = 1; i <= page; i++) {
            list.push(url + "/" + i);
        }

    } else {
        list.push(url.replace("(http|https)://truyenvkl.com", ""));
    }

    return Response.success(list)
}